linters/cmake-format/cmake-format.test.ts

```diff
@@ -0,0 +1,3 @@
+import { linterFmtTest } from "tests";
+
+linterFmtTest({ linterName: "cmake-format" });
```

Comment:
Depending on the outcome of the `lint` subcommand, it may be preferred to run `linterFmtTest` and `linterCheckTest` here.